RNTester/js/AccessibilityIOSExample.js

```diff
@@ -12,18 +12,20 @@
 
 const React = require('react');
 const ReactNative = require('react-native');
-const {AccessibilityInfo, Text, View, TouchableOpacity} = ReactNative;
+const {AccessibilityInfo, Text, View, TouchableOpacity, Alert} = ReactNative;
 
 class AccessibilityIOSExample extends React.Component<{}> {
   render() {
     return (
       <View>
         <View
-          onAccessibilityTap={() => alert('onAccessibilityTap success')}
+          onAccessibilityTap={() => Alert.alert('onAccessibilityTap success')}
```

Comment:
This is almost the same. global's alert sets the title of the alert to "Alert". https://github.com/facebook/react-native/blob/master/Libraries/Core/setUpAlert.js#L20